mac/Keyman4Mac/Keyman4Mac/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>Keyman $(PRODUCT_VERSION) for macOS</string>
+	<string>13.0.99</string>
```

Comment:
Does this string title get updated somewhere else? (with the new $(PRODUCT_VERSION))